### ISSUE TITLE
Chirumiru: Display Bug

### ISSUE THREAD

**Issue by p0008874** [ruffle-rs/ruffle#7621]
### Describe the bug

https://streamable.com/vh06bq
```
[2022-08-09T15:07:44Z ERROR ruffle_core::avm2::activation] AVM2 error: Property does not exist: Multiname { ns: [Package("flash.filters")], name: Some("BitmapFilterQuality"), params: [] }
[2022-08-09T15:07:44Z ERROR ruffle_core::display_object::movie_clip] Got Property does not exist: Multiname { ns: [Package("flash.filters")], name: Some("BitmapFilterQuality"), params: [] } when constructing AVM2 side of movie clip
[2022-08-09T15:07:44Z ERROR ruffle_core::avm2::activation] AVM2 error: Cannot access property ::perspectiveProjection of null or undefined
[2022-08-09T15:07:44Z ERROR ruffle_core::avm2::events] Error dispatching event EventObject(EventObject(GcCell(Gc { ptr: 0x224128df6c8 }))) to handler FunctionObject(FunctionObject(GcCell(Gc { ptr: 0x22412f1ad58 }))) : "Cannot access property ::perspectiveProjection of null or undefined"
[2022-08-09T15:09:18Z ERROR ruffle_core::display_object::movie_clip] Unable to instantiate display node id 460, reason being: Not a DisplayObject
[2022-08-09T15:09:24Z ERROR ruffle_core::avm2::activation] AVM2 error: Could not find script for class trait Multiname { ns: [Package("flash.geom")], name: Some("Matrix3D"), params: [] }
[2022-08-09T15:09:24Z ERROR ruffle_core::avm2::activation] AVM2 error: Could not find script for class trait Multiname { ns: [Package("flash.geom")], name: Some("Matrix3D"), params: [] }
[2022-08-09T15:09:24Z ERROR ruffle_core::avm2::activation] AVM2 error: Could not find script for class trait Multiname { ns: [Package("flash.geom")], name: Some("Matrix3D"), params: [] }
[2022-08-09T15:09:24Z ERROR ruffle_core::avm2::activation] AVM2 error: Could not find script for class trait Multiname { ns: [Package("flash.geom")], name: Some("Matrix3D"), params: [] }
[2022-08-09T15:09:24Z ERROR ruffle_core::display_object::movie_clip] Got Could not find script for class trait Multiname { ns: [Package("flash.geom")], name: Some("Matrix3D"), params: [] } when constructing AVM2 side of movie clip
[2022-08-09T15:09:34Z ERROR ruffle_core::display_object::movie_clip] Unable to instantiate display node id 564, reason being: Not a DisplayObject
```

### Expected behavior

https://streamable.com/5itgbd

### Affected platform

Desktop app

### Operating system

Windows 10

### Additional information

SWF file:
https://web.archive.org/web/20200121160532/http://www.l-room.net/product/movie/chirumiru.html

**Comment by p0008874** [comment]
Is seen this was fixed by #7570 
But some effect still not fixed.

**Comment by p0008874** [comment]
OUTDATED INFO.
